### DISCUSSION TITLE
performance issues with the number of ETCD connections

### DISCUSSION THREAD

**Discussion by chenws1012** [apache/apisix#5872] [Q&A]
看到博客里有提到  Apache APISIX 的每个 worker 都会跟 ETCD 连接，每一个监听目录都会去建一个连接。
这个官方后续会优化吗？

**Comment by tokers** [comment]
Yes, we'll continue to optimize it. BTW, do you have some good ideas to optimize it?

**Reply by tzssangglass** [reply]
yes, we have been trying to optimise this. The options I know of so far are
- 1 subscribe only to the `/apisix` prefix and differentiate the directories within apisix
- 2 add an etcd proxy between APISIX and etcd so that there is only 1 request to etcd

we have community meetings, would you like to talk about this?

**Reply by juzhiyuan** [reply]
see https://github.com/apache/apisix/discussions/5873
